workspace/api/openfdd_bridge/niagara_store.py

```
@@ -0,0 +1,395 @@
+    point_cols: dict[str, str] = {}
+    try:
+        from .timeseries_api import plot_column_name
+
+        model = ModelService().load()
+        for pt in model.get("points") or []:
+            if not isinstance(pt, dict):
+                continue
+            pid = str(pt.get("id") or "").strip()
+            if pid:
+                point_cols[pid] = plot_column_name(pt)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)